docs/extend/authorization.md

```diff
@@ -0,0 +1,120 @@
+<!--[metadata]>
++++
+title = "Access authorization plugin"
+description = "How to create authorization plugins to manage access control to your Docker daemon."
+keywords = ["security, authorization, authentication, docker, documentation, plugin, extend"]
+[menu.main]
+parent = "mn_extend"
+weight = -1
++++
+<![end-metadata]-->
+
+
+# Access authorization
+
+By default Docker’s authorization model is all or nothing. Anyone who has access to the Docker daemon has the ability to run any Docker command. Authorization plugins enable creating granular access policies for managing access to Docker daemon. 
```

Comment:
I wonder if we should refer to the docker *daemon* or the docker remote API; "Anyone who has access to the Docker remote API.."

The last sentence is quite long and a bit confusing; perhaps

```
Authorization plugins enable you to create granular access policies to the Docker remote API.
```